### ISSUE TITLE
Rotational Free Energy calculation

### ISSUE THREAD

**Issue by EffettiCollaterali** [mesonepigreco/CellConstructor#4]
Hi, 
it seems that the calculation of free energy could be improved. In particular, is not clear whether the symmetry number sigma is computed from the dynamical matrix or not. In any case the symmetries after the sscha calculation could be unprecise. Furthermore, it could be useful to print also the entropy and enthalpy contribution separately. 

http://www.codessa-pro.com/descriptors/thermodynamic/rem.htm

https://github.com/mesonepigreco/CellConstructor/blob/c39bf082637ffbeb3801e3e062db4fcb7b77c723/cellconstructor/Structure.py#L1910

**Comment by mesonepigreco** [comment]
Yes, as it is right now, the calculation of the classical rotational free energy is not done correctly, I think that I will disable for the master and the 1.0 Milestone, and maybe move it on a separate branch for a future version.
